### ISSUE TITLE
cardinal problem 

### ISSUE THREAD

**Issue by wajid404040** [pikaju/flutter-braintree#136]
Hello,

My app was recently rejected by Google Play because it was found to be using a version of the Braintree SDK that violates their policies.

The specific issue is with the Cardinal Commerce SDK that collects Android ID and Hardware IDs identifiers. Google Play's policy does not allow linking persistent device identifiers to other personal and sensitive user data or resettable device identifiers.

The solution to this problem is to update to version 6.9.0 or later of the Braintree SDK, as mentioned in the [Braintree Drop-in UI release notes](https://github.com/braintree/braintree-android-drop-in/releases/tag/6.9.0).

Could you please update the `flutter_braintree` package to use a compliant version of the Braintree SDK?

Thank you.

**Comment by pikaju** [comment]
Hey, please try version `4.0.0-dev.1` of `flutter_braintree`. It should fix this issue.

**Comment by wajid404040** [comment]
> Hey, please try version `4.0.0-dev.1` of `flutter_braintree`. It should fix this issue.

I have but the same error from google playstore 
" Your app is using the Cardinal Commerce SDK which collects but may not be limited to Android ID and Hardware IDs identifiers. Persistent device identifiers may not be linked to other personal and sensitive user data or resettable device identifiers.
You may consider moving to another SDK; removing this SDK; or if available from your SDK provider, upgrading to a policy-compliant version of this SDK that does not include the violating code.

According to the information provided by your SDK provider, you may consider upgrading to 2.2.7-2. Please consult the SDK provider for further information.

Affected Version(s):

App Bundle:2, App Bundle:2 "

**Comment by pikaju** [comment]
Oh okay, currently it is on drop-in version 6.8, could you try forking the repository and changing to 6.9? If it works, you can create a pull request and I'll publish a new version.

**Comment by wajid404040** [comment]
> Oh okay, currently it is on drop-in version 6.8, could you try forking the repository and changing to 6.9? If it works, you can create a pull request and I'll publish a new version.

Indeed, the update to version 6.9 was successful. For those encountering similar issues, it's crucial to ensure that the latest version of Android is being used for both internal ,closed and  open testing. Thank you.

**Comment by HarshSteyn** [comment]
Google play console says we should update the cardinal sdk version to 2.2.7-2 so where should i update this version
